src/models/activityReport.js

```diff
@@ -275,6 +275,20 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: true,
       type: DataTypes.ENUM(Object.keys(USER_ROLES).map((k) => USER_ROLES[k])),
     },
+    creatorName: {
+      type: DataTypes.VIRTUAL,
+      get() {
+        // If a creator role is not set but the report is in ALERTS.
+        // We want to show the name with no roles.
+        if (this.creatorRole || this.calculatedStatus !== REPORT_STATUSES.APPROVED) {
```

Comment:
I'm not 100% sold on the second half of this statement. Not sure if its better to hard code to (===) DRAFT only.

Reply:
Do we even need this check here? Can we just let `creatorNameWithRole` always get called and if the report doesn't have a role one isn't added?

Reply:
Will keep as is for now we will remove once we run a migration.